mitigate-log4shell.conf

```diff
@@ -21,6 +21,9 @@ local h, err = ngx.req.get_headers()
 for k, v in pairs(h) do
   req_headers = req_headers .. k .. ": " .. tostring(v) .. "\n";
   if v then
+    if string.match(tostring(v), "JHtqbm") then
```

Comment:
This has the potential to match all kind of random data (e.g. tokens). I'll try making another PR.